src/DGtal/shapes/Mesh.ih

```diff
@@ -283,6 +283,34 @@ DGtal::Mesh<TPoint>::getFaceColor(unsigned int i) const
     }
 }
 
+
+template<typename TPoint>
+inline
+std::pair<TPoint, TPoint>  
+DGtal::Mesh<TPoint>::getBoundingBox() const
+{
+  std::pair<TPoint, TPoint> theResult;  
+  std::vector<std::vector<typename TPoint::Component> > vectComp;
+  for(unsigned int i=0; i< TPoint::size(); i++){
+    vectComp.push_back(std::vector<typename TPoint::Component>());
+  }
+  for( typename DGtal::Mesh<TPoint>::VertexStorage::const_iterator it = vertexBegin(); 
+       it!= vertexEnd(); it++){
+    for(unsigned int i=0; i< TPoint::size(); i++){
+      vectComp[i].push_back((*it)[i]);
+    }
```

Comment:
why do you need to copy the vertex storage and not directly do the std::min_element on the VertexStorage?